src/flake.nix

```
@@ -45,9 +45,7 @@
       in {
         packages."${pkgs.system}" = {
           ci = pkgs.runCommand "ci" {} ("ls " + toString config.ci + " && touch $out");
-          procfile = config.procfile;
-          procfileEnv = config.procfileEnv;
-          info = config.info;
+          inherit (config) info procfileScript;
```

We need to keep these for backwards compatibility.

```suggestion
          inherit (config) info procfileScript procfileEnv procfile;
```